### PR TITLE
fx: fix db migrations

### PR DESCRIPTION
**Pull request by SergeInGithub**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

- Fixed db migrations where we used the sync method before to use migrations instead as it is the the recommended way to migrate tables in sequelize. 

Fixes # (issue)
#184361196 DB migrations

# Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or Enhancement (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have created a loom video for the new feature or enhancement 
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by musllim** [comment]
do you have scripts for running migrations?

**Review by shyakadavis — DISMISSED** [review]
LGTM.
But message to the others...
```
Watch for my P.R, because it has the refactored code base to account for the changes requested from Axel. e.g (uuids, increased. tests)
```

**Review by musllim — DISMISSED** [review]
goood

File: src/db/models/index.ts [hunk 1]
```diff
@@ -1,5 +1,62 @@
-export * from './Cart.model';
-export * from './LoggedInUsers.model';
-export * from './Product.model';
-export * from './Token.model';
-export * from './User.model';
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-undef */
+/* eslint-disable @typescript-eslint/no-var-requires */
```
Comment (freddykaberuka) [inline src/db/models/index.ts:3]: - [ ] Kindly remove this & solve the linter manually

**Review by freddykaberuka — CHANGES_REQUESTED** [review]
Hi @SergeInGithub, Good job so far! 
There are some feedbacks from Bot that you still need to work on, but you are almost there!

**Comment by SergeInGithub** [comment]
> Hi @SergeInGithub, Good job so far!
> There are some feedbacks from Bot that you still need to work on, but you are almost there!

- The sourcetype is module, it's perfectly set to module in eslint, i don't know why it's failing

File: src/db/models/index.ts (hunk 1 above)
Comment (SergeInGithub) [reply src/db/models/index.ts:3]: i've tried removing these and manually adding it in the eslint file but it doesn't work, that's why i add it in manually

File: test/auth.spec.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import request from 'supertest';
 import app from '../src/app';
 import { sequelize } from '../src/db/config';
+jest.setTimeout(1500000);
```
Comment (mucyoalexaxel) [inline test/auth.spec.ts:4]: why this timeout ?

**Pull request closed without merging**